src/components/common/YourCompany/BusinessContactInfo.vue

```diff
@@ -6,7 +6,6 @@
     :hasBusinessContactInfoChange="hasBusinessContactInfoChange"
     :editLabel="editLabel"
     :editedLabel="editSavedLabel"
-    :disableActions="isCorrectionFiling"
```

Comment:
Is this for ticket 13776?

Reply:
Yes it looks like it, I just did it for 13776

Reply:
Ticket 13776 overlaps with this ticket, and I mentioned it in planning yesterday.

Reply:
How are you both working on the same ticket? You should always put your name on the tickets you're working on.

Reply:
I'll close 13776

Reply:
> How are you both working on the same ticket? You should always put your name on the tickets you're working on, to prevent this.

Different tickets but they had an overlap with each other